src/main.js

```diff
@@ -158,22 +152,22 @@ function getRandomIndex(array) {
 function saveCurrentPoster() {
```

Comment:
You can move the savedPostersGrid query selector variable inside the saveCurrentPoster function as it is a local variable.

Reply:
I went ahead and made this update!